sdk/servicebus/service-bus/README.md

```diff
@@ -102,7 +102,7 @@ using the [createSender][sbclient_createsender] method.
 This gives you a sender which you can use to [send][sender_send] messages.
 
 ```javascript
-const sender = await serviceBusClient.createSender("my-queue");
+const sender = serviceBusClient.createSender("my-queue");
```

Comment:
Not sure where to leave this comment, but I noticed the API version referenced at the top of the readme is preview.2. I just want to make sure it gets changed to preview.3 before being released (and since this change only works in preview.3...seemed reasonable to maybe change the version here :) )

Reply:
Yes, I was thinking of doing a separate PR for that since it immediately breaks all the docs for anyone visiting the repo.

Reply:
Sounds good!